sdk/lib/core/double.dart

```diff
@@ -123,13 +123,13 @@ abstract final class double extends num {
   /// Rounds away from zero when there is no closest integer:
   ///  `(3.5).roundToDouble() == 4` and `(-3.5).roundToDouble() == -4`.
   ///
-  /// If this is already an integer valued double, including `-0.0`, or it is not
-  /// a finite value, the value is returned unmodified.
+  /// If this is already an integer valued double, including `-0.0`, or it is
+  /// not a finite value, the value is returned unmodified.
```

Comment:
This is the only line which is too long, the rest do not need to be reflowed.
And I don't think most of them *should* be.